### ISSUE TITLE
train DDSC failed

### ISSUE THREAD

**Issue by smajida** [GeorgeSeif/Semantic-Segmentation-Suite#115]
2. Bug reports

------------------------

## Please specify the following information when submitting an issue:
- **What are your command line arguments?:** python train.py --dataset Vehicle_512 --model DDSC
- **Have you written any custom code?**: no not at all. just using the fresh code.
- **What have you done to try and solve this issue?:** yeah, searched a lot to find the error. some people say the problem is with the cuda version. i tried different cuda versions, 8.0, 9.0,9.1, 9.2 none worked.
- **TensorFlow version?**: 1.11
### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug or a feature request.
choosing the DDSC algorithm throws the following error:

```
2018-10-09 00:29:11.162019: F tensorflow/stream_executor/cuda/cuda_dnn.cc:430] could not convert BatchDescriptor {count: 1 feature_map_count: 1024 spatial: 0 0  value_min: 0.000000 value_max: 0.000000 layout: BatchDepthYX} to cudnn tensor descriptor: CUDNN_STATUS_BAD_PARAM
Aborted (core dumped)

```

**Comment by GeorgeSeif** [comment]
Yup I know! Still trying to figure out what that bug means.... Thanks for opening up the issue though!

**Comment by smajida** [comment]
@GeorgeSeif I am also trying to figuring it out. 
as small side question, is the issue with InceptionV4 resolved? no matter which algorithm I choose, selecting InceptionV4 as backbone fails.

**Comment by GeorgeSeif** [comment]
Apologies for the late reply!

For DDSC I always had trouble, but left the code in there for reference. So I haven't fully trained it yet :)
